### ISSUE TITLE
Using t2d, when any message is sent in Telegram the script exits on "Cannot read property send of undefined"

### ISSUE THREAD

**Issue by HotdogCombo** [TediCross/TediCross#129]
The Telegram token is set (The code with : in it), the Discord token is set (The code with . in it), and the channel ID is set with single quotes '##########'.

Error Log:

> 2020-01-23 15:54:45 [INFO] Discord: DisBot. (################)
> 2020-01-23 15:54:46 [INFO] Telegram: TelBot (#########)
> (node:15758) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'send' of undefined
>     at R.forEach (/home/user/TediCross/src/telegram2discord/endwares.js:137:33)
>     at forEach (/home/user/TediCross/node_modules/ramda/src/forEach.js:45:5)
>     at /home/user/TediCross/node_modules/ramda/src/internal/_checkForMethod.js:22:72
>     at /home/user/TediCross/node_modules/ramda/src/internal/_curry2.js:22:18
>     at f1 (/home/user/TediCross/node_modules/ramda/src/internal/_curry1.js:18:17)
>     at relayMessage (/home/user/TediCross/src/telegram2discord/endwares.js:144:4)
>     at execute (/home/user/TediCross/node_modules/telegraf/composer.js:370:13)
>     at Promise.resolve.handler (/home/user/TediCross/node_modules/telegraf/composer.js:370:49)
>     at execute (/home/user/TediCross/node_modules/telegraf/composer.js:370:13)
>     at Promise.resolve.handler (/home/user/TediCross/node_modules/telegraf/composer.js:370:49)
> (node:15758) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
> (node:15758) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

Also, the script is not bringing the Discord bot online, using Discord.js 11.5.1. Which i'm guessing is what's causing this issue, since the Telegram portion can see there's a new message but can't connect to the Discord side to send it?

**Comment by Suppen** [comment]
This is a cpmmon error which is aømost always caused by putting the wrong channel ID in the settings file. Please verify you have the correct ID by sending `/chatinfo` in the channel in question. Remember the auotes around the ID when you pu it into the settings file

**Comment by HotdogCombo** [comment]
Sorry, this is a non-issue as I have just discovered.

I moved to a new discord account and had the bot still in my server from the old account which had a different token than what I was copying on my new discord developer account. 

The bot would never come online because it was trying to enable the token for the new bot when only the old was in the server. Since they shared the same name I didn't immediately notice.

Just kicked out the old version and invited the new one, it worked immediately!

Thank you for the quick response.
